MetaMask/metamask-extension
# Update `object.values` patch

The package `object.values` was updated in #15293 but we forgot to update the patch. This was resulting in a warning on the command line upon each install.

The patch was still applied successfully, so no changes were needed other than updating the version.

## Manual Testing Steps

Ensure there is no CLI warning upon install. The warning looks like this:

```
Warning: patch-package detected a patch file version mismatch                                     
                                                                                                  
  Don't worry! This is probably fine. The patch was still applied                                 
  successfully. Here's the deets:                                                                 
                                                                                                  
  Patch file created for                                                                          
                                                                                                  
    object.values@1.1.3                                                                           
                                                                                                  
  applied to                                                                                      
                                                                                                  
    object.values@1.1.5                                                                           
                                                                                                  
  At path                                                                                         
                                                                                                  
    node_modules/object.values                                                                    
                                                                                                  
  This warning is just to give you a heads-up. There is a small chance of                         
  breakage even though the patch was applied successfully. Make sure the package                  
  still behaves like you expect (you wrote tests, right?) and then run                            
                                                                                                  
    patch-package object.values
                                                                                                  
  to update the version in the patch file name and make this warning go away.                     
                                                                                                  
---                                                                                               
patch-package finished with 1 warning(s).        

```

## Pre-Merge Checklist


### + If there are functional changes:
